### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [jeffalexhickman/TeaCozy#2]
So our courses don't talk about accessibility enough.  Don't consider this a critique but an instruction on future practices.  Each image should have a descriptive alt text.  What happens is that screen readers for the visually impaired i.e. blind will read the al text out loud for them when they hover over the image.
